app/assets/stylesheets/mail/email.scss

```diff
@@ -63,6 +63,12 @@ p.notice {
   margin-top: 20px;
 }
 
+p.powered-by {
+  background-color: #ebebeb;
+  padding: .5em;
```

Comment:
I was wondering if `em` unit is correctly handled by emailer. Seems so : https://www.caniemail.com/features/css-unit-em/ !

Reply:
nice! I didn't know caniemail.com :heart_eyes: